docs/administration.md

```diff
@@ -0,0 +1,72 @@
+Administering TicTacToe
+=======================
+
+Some of the details here overlap with the develop documentation, so you might want to take
+a look at that as well.
+
+# Prerequisites
+
+We use Java 8 features, so ensure that you have Java 8 runtime installed.
+You can find it here: http://www.oracle.com/technetwork/java/javase/downloads/jre8-downloads-2133155.html
+
+# Releases
+
+Our releases can be found on [our GitHub releases page](https://github.com/hundarogborn/TicTacToe/releases).
+
+Each release is simply an "uberjar", a self-contained JAR for minimal overhead, so you do not need to install
+any dependencies.
+
+# Running
+
+TicTacToe can be run in two ways:
+
+- As a command line application where you interactively play
+
+- As a web application
+
+## Running the CLI app
+
+Download an uberjar from our release page and:
```

Comment:
Maybe we could put the link to our release page and instead of saying "+Download an uberjar from our release page and:" both in Running the CLI app and in Running the Web app have it in the Running section above?